webapp/components/player/player.tsx

```diff
@@ -210,7 +209,13 @@ export default function Player(props: { stream: MediaStream, muted: boolean, aud
   // https://webkit.org/blog/6784/new-video-policies-for-ios/
   return (
     <center className="relative flex flex-col justify-center min-h-60 rounded-xl bg-black m-8" style={{ width: props.width }}>
-      {!props.stream.getTracks().length ? <center><SvgProgress /></center> : null}
+      {!props.stream.getTracks().length ?
+        <p
+          className={`text-white transition-opacity duration-300 ${
+            showControls ? 'opacity-100' : 'opacity-0'
+          }`}>
+          All your media are disabled
```

Comment:
Maybe this message should be different between whep-player and whip-player? 

"All your media" refers to the current user, it may be confusing when this tip shows on other user's stream.

Maybe we keep the SvgProgress element on WhepPlayer, only show this tip on Prepare page and WhipPlayer?